plugins/bastion-paper/src/main/java/isaac/bastion/BastionBlock.java

```diff
@@ -95,13 +96,7 @@ public double getErosionFromElytra() {
      */
     public boolean inField(Location loc) {
         return !(yLevelCheck(loc) || // don't add parens if you don't know why it wasn't there.
-            (type.isSquare() &&
-                (Math.abs(loc.getBlockX() - location.getBlockX()) > type.getEffectRadius() ||
-                    Math.abs(loc.getBlockZ() - location.getBlockZ()) > type.getEffectRadius())) ||
-            (!type.isSquare() &&
-                ((loc.getBlockX() - location.getX()) * (float) (loc.getBlockX() - location.getX()) +
-                    (loc.getBlockZ() - location.getZ()) * (float) (loc.getBlockZ() - location.getZ()) >= type.getRadiusSquared()))
-        );
+            (type.isSquare() && (Math.abs(loc.getBlockX() - location.getBlockX()) > type.getEffectRadius() || Math.abs(loc.getBlockZ() - location.getBlockZ()) > type.getEffectRadius())) || (!type.isSquare() && ((loc.getBlockX() - location.getX()) * (float) (loc.getBlockX() - location.getX()) + (loc.getBlockZ() - location.getZ()) * (float) (loc.getBlockZ() - location.getZ()) >= type.getRadiusSquared())));
```

Comment:
why remove the formatting that made this readable?

Reply:
Yea your right, auto formatting error. I'll try and force the formatter to ignore.